components/sliders.css

```diff
@@ -0,0 +1,46 @@
+.slidecontainer {
+    width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
use the properties as helper class and include all 3 main proerties in one class and then use it

Reply:
Okay! Will do it.